express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+const newArray = [];
+app.get('/api/grades', (req, res) => {
+  for (const prop in grades) {
+    newArray.push(grades[prop]);
+  }
+  res.json(newArray);
+})
```

Comment:
The array should be newly created with every request. As it is grades will keep getting added to the same array with every request so the array will continue to get larger.